virttest/bootstrap.py

```diff
@@ -351,6 +351,10 @@ def _forced_or_detected(forced, detected):
                                       "u%s" % detected.release)
         arch = _forced_or_detected(get_opt(options, 'vt_host_distro_arch'),
                                    "Host_arch_%s" % detected.arch)
+        vendor = cpu.get_vendor() if hasattr(cpu, 'get_vendor') else cpu.get_cpu_vendor_name()
+        family = cpu.get_family() if hasattr(cpu, 'get_family') else None
```

Comment:
@sathnaga   Wouldn't it failed for amd as `NotImplementedError`
https://github.com/avocado-framework/avocado/blob/25bc654cc254015cfb5d2284db7a2c9e15223acc/avocado/utils/cpu.py#L187

I hit this issue now, do i miss any other dependent patch or it is a regression issue?   Thanks!

cc @luckyh @nanliu-r

Reply:
@yanan-fu I believe all the dependencies are merged unless you are using older Avocado, there are some discussion going on about it, pls refer this, https://github.com/avocado-framework/avocado-vt/pull/2455#issuecomment-666220878

can you explain more on your issue, so you run on AMD and hit the issue? pls paste more logs
Thanks!

Reply:
@yanan-fu ah, I see the issue, get.family() needs a handle for `NotImplementedError`, will send a quick fix.. confirm if that's what you are referring...

Reply:
> @yanan-fu ah, I see the issue, get.family() needs a handle for `NotImplementedError`, will send a quick fix.. confirm if that's what you are referring...

Ok, I think we are in the same line now.   And here is the traceback:
```
Avocado crashed:
Traceback (most recent call last):
  File "/home/kar/workspace/bin/avocado", line 33, in <module>
    sys.exit(load_entry_point('avocado-framework==80.0', 'console_scripts', 'avocado')())
  File "/home/kar/workspace/avocado/avocado/core/main.py", line 76, in main
    return app.run()
  File "/home/kar/workspace/avocado/avocado/core/app.py", line 97, in run
    return method(self.parser.config)
  File "/home/kar/workspace/lib/python3.6/site-packages/avocado_framework_plugin_vt-80.0-py3.6.egg/avocado_vt/plugins/vt_bootstrap.py", line 99, in run
    bootstrap.bootstrap(options=config, interactive=True)
  File "/home/kar/workspace/lib/python3.6/site-packages/avocado_framework_plugin_vt-80.0-py3.6.egg/virttest/bootstrap.py", line 956, in bootstrap
    create_host_os_cfg(options)
  File "/home/kar/workspace/lib/python3.6/site-packages/avocado_framework_plugin_vt-80.0-py3.6.egg/virttest/bootstrap.py", line 355, in create_host_os_cfg
    family = cpu.get_family() if hasattr(cpu, 'get_family') else None
  File "/home/kar/workspace/avocado/avocado/utils/cpu.py", line 188, in get_family
    raise NotImplementedError
NotImplementedError
```

Looking forward to your hot fix~   thanks @sathnaga

Reply:
Oups, didn't catch that one. Sorry. Should also fail on arm, right... Feel free to ping me for a review (btw wouldn't it be better to simply return "Unknown" instead of raising an exception on the Avocado side?)

Reply:
@yanan-fu pls let me know if this fixes your issue, https://github.com/avocado-framework/avocado-vt/pull/2696

Reply:
Thanks for reporting!

Reply:
> Oups, didn't catch that one. Sorry. Should also fail on arm, right... Feel free to ping me for a review (btw wouldn't it be better to simply return "Unknown" instead of raising an exception on the Avocado side?)


@ldoktor my bad, did not realise it earlier, yes Arm also should fail with the same error.
just handled it here, #2696, pls review Tnx!
handled similar issue with vendor aswell.